### PR TITLE
prefix always used

### PR DESCRIPTION
**Pull request by kirstyellis**
linked to #24 in sr_config

### CODE REVIEW

**Comment by guihomework** [comment]
@kirstyellis , for tuning (according to your team) and use of a single hand (UPMC), one would like to not have any prefix.  
of course, now almost all of the GUI tools handle prefixes.

Does this mean that all the non-prefixed files will be removed too ?

**Comment by ugocupcic** [comment]
@guihomework yes. We're working on homogenizing our configs. The best way we found of doing this is to use prefix all the time. Of course, we'll finish the migration of the gui tools as well (using [this](https://github.com/shadow-robot/sr-ros-interface/issues/343))

File: sr_edc_launch/sr_edc.launch [hunk 1]
```diff
@@ -18,8 +18,6 @@
        In case we only have one hand this parameter will normally be "", meaning that the desired params are in the root of those directories
        If it's not "" then it must be followed by a "/" e.g. "hand_2/"  -->
   <arg name="config_dir" default=""/>
```
Comment (ugocupcic) [inline sr_edc_launch/sr_edc.launch:20]: so shouldn't this default to `rh`?

Comment (kirstyellis) [inline sr_edc_launch/sr_edc.launch:20]: yeah you're right, but in my next PR I am removing this parameter so I guess I missed it :-/

**Comment by ugocupcic** [comment]
Closing this, we'll group it with the other issue about mapping.

**Pull request closed without merging**